go/performance/scripts/dg_sysbench.sh

```diff
@@ -0,0 +1,145 @@
+#!/bin/bash
+set -e
+set -o pipefail
+
+SYSBENCH_TEST="oltp_insert_only"
+WORKING_DIR=`mktemp -d`
+PPROF=0
+PORT=5433
+
+# parse options
```

Comment:
Might add a top-level comment on what this script is for and some usage examples